nh_clinical/tests/user_allocation/helpers.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+def create_test_data(cls, cr, uid, iterations=3):
```

Comment:
Yeah sure, as discussed I'll move these tests into the demo data module and use the data there to speed this up and make it less of a maintenance headache.